lib/rules/no-const-assign.js

```diff
@@ -13,6 +13,8 @@ const astUtils = require("../util/ast-utils");
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
This seems like it should be a `problem`; the patterns reported by this rule will generally throw an error at runtime.

Reply:
Agreed